sunpy/net/base_client.py

```diff
@@ -397,6 +397,7 @@ def __init_subclass__(cls, *args, **kwargs):
 
         cls._registry[cls] = cls._can_handle_query
 
+        # If a client has an _attrs_module() classmethod, import that module's contents into the attrs namespace.
```

Comment:
I don't think this comment should be here.

Reply:
alright :+1: